#### Мат.стат Урок 1.



In [5]:
from math import factorial
def combinations(n, k):
    return int(factorial(n) / (factorial(k) * factorial(n - k)))
def arrangements(n, k):
    return int(factorial(n) / factorial(n - k))
def permutations(n):
    return int(factorial(n))

1. Из колоды в 52 карты извлекаются случайным образом 4 карты. a) Найти вероятность того, что все карты – крести. б) Найти вероятность, что среди 4-х карт окажется хотя бы один туз.

**a)** Верояность достать первой картой крести = 13 / 52, второй = 12 / 51, третей = 11 / 50, четвёртой = 10 / 49. Умножаем вероятность.

In [13]:
(13/52) * (12/51) * (11/50) * (10/49)

0.0026410564225690276

In [20]:
a = combinations(13,4)
a

715

In [21]:
b = combinations(52,4)
b

270725

In [22]:
a/b

0.0026410564225690276

**Ответ:** 0.0026410564225690276

**б)** вероятность, что среди 4-х карт окажется хотя бы один туз.

In [23]:
c = combinations(48,4)
c

194580

In [24]:
1 - (c/b)

0.2812632745405854

**Ответ:** 0.2812632745405854

2. На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9. Код содержит три цифры, которые нужно нажать одновременно. Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?

In [26]:
combinations(10,3)

120

**Ответ:** $$\frac{1}{120}$$

3. В ящике имеется 15 деталей, из которых 9 окрашены. Рабочий случайным образом извлекает 3 детали. Какова вероятность того, что все извлеченные детали окрашены?

In [27]:
(9/15)*(8/14)*(7/13)

0.18461538461538457

In [29]:
a1 = combinations(15,3)
b1 = combinations(9,3)
b1/a1

0.18461538461538463

**Ответ:** 0.1846154

4. В лотерее 100 билетов. Из них 2 выигрышных. Какова вероятность того, что 2 приобретенных билета окажутся выигрышными?

In [33]:
a2 = combinations(100,2)
b2 = combinations(2,2)
print(a2,b2,b2/a2)

4950 1 0.00020202020202020202


**Ответ:** 0.00(02)